### DISCUSSION TITLE
16+ event: add variable to tickets

### DISCUSSION THREAD

**Discussion by wardverduyn** [pretix/pretix#4505] [Q&A]
We have an event which is 16+, so they need to be at least 16 years old to enter our event. Is it possible to ask for the birth date of every visitor of our event who buys ticket (so this should be filled in per ticket, not per buyer). If possible I would like to add to their ticket if they are 16+ or 18+ based on their birth date. If possible I would like to do this in this way, not by adding two different tickets. (We will still check their ID at the event itself if they lie about their age)

**Comment by raphaelm** [comment]
It is not possible to do a "if-this-than-that" logic on ticket layouts, sorry
